server/internal/infrastructure/tx-builder/covenant/builder.go

```diff
@@ -1071,7 +1087,11 @@ func (b *txBuilder) VerifyAndCombinePartialTx(dest string, src string) (string,
 		}
 	}
 
-	return roundSigner.Pset.ToBase64()
+	b64, err := roundSigner.Pset.ToBase64()
```

Comment:
```suggestion
	return roundSigner.Pset.ToBase64()
```
what was the problem with this ?